src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -557,6 +579,38 @@ bool fuse_type_to_parameter(const std::shared_ptr<ov::Node>& node,
     return changed;
 }
 
+bool wrap_into_original_type(const std::shared_ptr<ov::Node>& node, const precisions_map& precisions) {
+    auto it = precisions.find(node->get_output_element_type(0));
+    if (it == precisions.end())
+        return false;
+
+    const auto& to = it->second;
+    const auto& from = it->first;
+
+    auto convert_before = std::make_shared<opset4::Convert>(node->input_value(0), from);
```

Comment:
Is this ok to use opset4 here? Should we use something about "ov::op::v0::Convert" instead?

Reply:
minor.
I will update opset/version in the next PR